tests/run_auth_tests.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
```

Comment:
Allows to run a subset of the tests, related to authentication. This list is not up to date, so getting rid of it.